File: generators/service-languageTranslator/templates/node/instrumentation.js
var IBMCloudEnv = require('ibm-cloud-env');
var LanguageTranslatorV2 = require('watson-developer-cloud/language-translator/v2');

module.exports = function(app, serviceManager){
    var languageTranslator = new LanguageTranslatorV2({
	    url: IBMCloudEnv.getString('languageTranslator_url'),
	    username: IBMCloudEnv.getString('languageTranslator_username'),
        password: IBMCloudEnv.getString('languageTranslator_password')
    });
    serviceManager.set("watson-language-translator", languageTranslator);
};

File: generators/service-documentConversion/templates/python/instrumentation.py
from ibmcloudenv import IBMCloudEnv
from watson_developer_cloud import DocumentConversionV1

document_conversion = DocumentConversionV1(
    username=IBMCloudEnv.getString('documentConversion_username'),
    password=IBMCloudEnv.getString('documentConversion_password'),
    version='2016-02-10')

def getService(app):
    return 'watson-document-conversion', document_conversion